client/src/components/FAQPanel/FAQPanel.js

```diff
@@ -0,0 +1,72 @@
+import _ from "lodash";
+import React from "react";
+
+import { SomeThingsToKeepInMind } from "src/panels/panel_declarations/common_panel_components";
+
+import common_lang from "src/panels/panel_declarations/misc/key_concept_panels/common_questions.yaml";
+
+import fin_lang from "src/panels/panel_declarations/misc/key_concept_panels/financial_questions.yaml";
+import ppl_lang from "src/panels/panel_declarations/misc/key_concept_panels/people_questions.yaml";
+import results_lang from "src/panels/panel_declarations/misc/key_concept_panels/results_questions.yaml";
+import tag_lang from "src/panels/panel_declarations/misc/key_concept_panels/tagging_questions.yaml";
+
+import sample_lang from "src/components/FAQPanel/FAQPanel.yaml";
+import {
+  create_text_maker_component,
+  KeyConceptList,
+} from "src/components/index";
+
+import { Gov } from "src/models/organizational_entities";
+
+import common_subapp_lang from "src/common_text/common_questions.yaml";
+
+import est_lang from "src/EstimatesComparison/estimates_comparison_questions.yaml";
+import igoc_lang from "src/IgocExplorer/igoc_explorer_questions.yaml";
+import table_picker_lang from "src/rpb/table_picker_questions.yaml";
+import tag_exp_lang from "src/TagExplorer/tag_explorer_questions.yaml";
+import text_diff_lang from "src/TextDiff/text_diff_questions.yaml";
+import treemap_lang from "src/TreeMap/treemap_questions.yaml";
```

Comment:
I do wonder if all of these yaml should co-exist with FAQPanel.js. Any thoughts on that?

Reply:
I think that's a good idea. I guess there isn't much point to keep them in their folders with the subapp